mkosi/__init__.py

```diff
@@ -566,10 +569,23 @@ def script_maybe_chroot_sandbox(
     options: Sequence[PathString],
     network: bool,
 ) -> Iterator[list[PathString]]:
-    options = ["--dir", "/work/src", "--chdir", "/work/src", *options]
+    localoptions = ["--dir", "/work/src", "--chdir", "/work/src"]
+
+    if parse_boolean(context.config.environment.get("MKOSI_CHROOT_SUPPRESS_CHOWN", "0")):
+        localoptions += ["--suppress-chown"]
```

Comment:
So this approach actually means we pass --suppress-chown to the non-chroot script as well. So we should handle this separately after all.

So I'd drop `localoptions` again and handle it like you did earlier after all, my bad.